pkg/identityserver/store/store_interfaces.go

```diff
@@ -65,6 +65,7 @@ type EndDeviceStore interface {
 	GetEndDevice(ctx context.Context, id *ttnpb.EndDeviceIdentifiers, fieldMask FieldMask) (*ttnpb.EndDevice, error)
 	UpdateEndDevice(ctx context.Context, dev *ttnpb.EndDevice, fieldMask FieldMask) (*ttnpb.EndDevice, error)
 	DeleteEndDevice(ctx context.Context, id *ttnpb.EndDeviceIdentifiers) error
+	BatchUpdateEndDevice(ctx context.Context, devs []*ttnpb.EndDevice, updateField string) ([]*ttnpb.EndDevice, error)
```

Comment:
I actually think it would be better to just add a `BatchUpdateEndDeviceLastSeen` instead of some generic `BatchUpdateEndDevice` that in the current implementation only supports `last_seen`.